docs: update custom domain docs (#4020)
FusionAuth/fusionauth-site: astro/src/content/docs/get-started/run-in-the-cloud/cloud.mdx

## What Does FusionAuth Cloud Cost

For pricing information, visit [the pricing page](/pricing).

### The Pricing Calculator

> You can also find the expected cost without creating an account by using the [pricing calculator](https://account.fusionauth.io/price-calculator).

Reviewer: ```suggestion
To find the expected cost without creating an account, use the [pricing calculator](https://account.fusionauth.io/price-calculator).
```
why use more word when few word do trick
